web/htdocs/assets/js/codesearch_ui.js

```diff
@@ -49,6 +49,16 @@ var Match = Backbone.Model.extend({
   initialize: function() {
   },
   url: function() {
+    if (this.get('tree') in CodesearchUI.intervalViewRepos) {
+      return this.internalUrl();
+    } else {
+      return this.externalUrl();
+    }
+  },
+  internalUrl: function() {
+    return "/view/" + this.get('tree') + "/" + this.get('path') + "#L" + this.get('lno');
```

Comment:
If we have a version hash, we should be embedding it here, so that this link, if copied/pasted, continues to point to the right line later on.